llm-finetune/train.py

```diff
@@ -210,11 +212,24 @@ def log_model_as_pipeline(
         tokenizer=training_arguments.output_dir,
         trust_remote_code=True,
     )
+    
+    hf_cache_info = scan_cache_dir()
+    files_to_save = []
+    for repo in hf_cache_info.repos:
+        if repo.repo_id == hf_hub_model_id:
+            for revision in repo.revisions:
+                for file in revision.files:
+                    if file.file_path.name.endswith(".py"):
+                        files_to_save.append(file.file_path)
+                break
```

Comment:
This needs one dedent
```suggestion
            break
```